lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -595,7 +595,7 @@ sub ia_base :Chained('base') :PathPart('view') :CaptureArgs(1) {  # /ia/view/cal
     my ( $self, $c, $answer_id ) = @_;
 
     $c->stash->{ia_page} = "IAPage";
-    $c->stash->{ia} = $c->d->rs('InstantAnswer')->find({meta_id => $answer_id});
+    $c->stash->{ia} = $c->d->rs('InstantAnswer')->find({meta_id => lc $answer_id});
```

Comment:
You can do something like this to be extra sure.

https://github.com/duckduckgo/community-platform/blob/65368f8/script/ddgc_revokeadmin.pl#L18

```perl
my $user = $schema->resultset('User')->search( \[ 'LOWER(me.username) = ?', ( lc($username) ) ] )->one_row;
```

Thanks!